src/proto_parser.cc

```diff
@@ -262,18 +262,19 @@ LIBRARY_EXPORT int LVGetServices(grpc_labview::LVProtoParser* parser, grpc_labvi
     }
 
     auto count = parser->m_FileDescriptor->service_count();
-    if (NumericArrayResize(0x08, 1, services, count * sizeof(ServiceDescriptor*)) != 0)
+    auto elementSize = sizeof(ServiceDescriptor*);
+    if (NumericArrayResize(grpc_labview::GetTypeCodeForSize(elementSize), 1, services, count * elementSize) != 0)
```

Comment:
NOTE: All of the changes to this file are just white space changes other than the five calls to `NumericArrayResize` in this file. They now all calculate the pointer size rather than always allocating 8 bytes for a pointer. I'm less certain about these changes since I'm a little confused as to why we are storing pointers to non LV data structures via array handles allocated by the LV memory manager. I'm also not certain who is calling these exported functions. The APIs appear to be related to grpc server reflection, but I don't know where/how they are being called.